bmi_lstm.py

```diff
@@ -76,7 +83,7 @@ def __init__(self):
     #------------------------------------------------------
     # Create a Python dictionary that maps CSDMS Standard
     # Names to the units of each model variable.
-    #------------------------------------------------------
+    #-------------------------_var_name_map_long_first-----------------------------
```

Comment:
I have a new dictionary (_var_name_units_map) which associates the long CSDMS standard name with the short model name and the units.